src/pykeen/contrib/lightning.py

```diff
@@ -247,12 +247,12 @@ def _dataloader(
 
 @click.command()
 @lit_module_resolver.get_option("-tl", "--training-loop")
-@dataset_resolver.get_option("-d", "--dataset", default="nations")
+@dataset_resolver.get_option("-ds", "--dataset", default="nations")
```

Comment:
i'm not really a fan of multiple letter short flags (this can get messy depending on if your shell interprets them as two individual flags)

Reply:
removed in fce3810